### ISSUE TITLE
Unit test LocalSlotMappingTests.Switch_Patterns broken in master

### ISSUE THREAD

**Issue by gafter** [dotnet/roslyn#19682]
The unit test `LocalSlotMappingTests.Switch_Patterns` is broken in master.

``` none
------ Test started: Assembly: Roslyn.Compilers.CSharp.Emit.UnitTests.dll ------

Test 'Microsoft.CodeAnalysis.CSharp.EditAndContinue.UnitTests.LocalSlotMappingTests.Switch_Patterns' failed: PDB format: Windows
Expected
====
@"<symbols>
  <methods>
    <method containingType=""C"" name=""M"">
      <customDebugInfo>
        <forward declaringType=""C"" methodName=""F"" />
        <encLocalSlotMap>
          <slot kind=""35"" offset=""11"" />
          <slot kind=""35"" offset=""11"" />
          <slot kind=""35"" offset=""11"" />
          <slot kind=""0"" offset=""106"" />
          <slot kind=""0"" offset=""162"" />
          <slot kind=""0"" offset=""273"" />
          <slot kind=""0"" offset=""323"" />
          <slot kind=""1"" offset=""11"" />
          <slot kind=""temp"" />
          <slot kind=""temp"" />
        </encLocalSlotMap>
      </customDebugInfo>
    </method>
  </methods>
</symbols>"

First Difference
====
Expected Fragment
----
<method containingType="C" name="M">
  <customDebugInfo>
    <forward declaringType="C" methodName="F" />
    <encLocalSlotMap>
      <slot kind="35" offset="11" />
      <slot kind="35" offset="11" />
      <slot kind="35" offset="11" />
      <slot kind="0" offset="106" />
      <slot kind="0" offset="162" />
      <slot kind="0" offset="273" />
      <slot kind="0" offset="323" />
      <slot kind="1" offset="11" />
      <slot kind="temp" />
      <slot kind="temp" />
    </encLocalSlotMap>
  </customDebugInfo>
</method>

Actual Fragment
----
<method containingType="C" name="M">
  <customDebugInfo>
    <forward declaringType="C" methodName="F" />
    <encLocalSlotMap>
      <slot kind="35" offset="11" />
      <slot kind="35" offset="11" />
      <slot kind="35" offset="11" />
      <slot kind="0" offset="106" />
      <slot kind="0" offset="162" />
      <slot kind="0" offset="273" />
      <slot kind="0" offset="323" />
      <slot kind="1" offset="11" />
      <slot kind="temp" />
    </encLocalSlotMap>
  </customDebugInfo>
</method>

Actual
====
@"<symbols>
  <methods>
    <method containingType=""C"" name=""M"">
      <customDebugInfo>
        <forward declaringType=""C"" methodName=""F"" />
        <encLocalSlotMap>
          <slot kind=""35"" offset=""11"" />
          <slot kind=""35"" offset=""11"" />
          <slot kind=""35"" offset=""11"" />
          <slot kind=""0"" offset=""106"" />
          <slot kind=""0"" offset=""162"" />
          <slot kind=""0"" offset=""273"" />
          <slot kind=""0"" offset=""323"" />
          <slot kind=""1"" offset=""11"" />
          <slot kind=""temp"" />
        </encLocalSlotMap>
      </customDebugInfo>
    </method>
  </methods>
</symbols>"


Expected: True
Actual:   False
	at Roslyn.Test.Utilities.AssertXml.Equal(XElement expectedRoot, XElement actualRoot, String message, String expectedValueSourcePath, Int32 expectedValueSourceLine, Boolean expectedIsXmlLiteral)
	at Microsoft.CodeAnalysis.Test.Utilities.PdbValidation.VerifyPdbImpl(Compilation compilation, IMethodSymbol debugEntryPoint, String qualifiedMethodName, String expectedPdb, DebugInformationFormat format, PdbToXmlOptions options, Int32 expectedValueSourceLine, String expectedValueSourcePath, Boolean expectedIsXmlLiteral)
	at Microsoft.CodeAnalysis.Test.Utilities.PdbValidation.VerifyPdb(Compilation compilation, String qualifiedMethodName, String expectedPdb, IMethodSymbol debugEntryPoint, DebugInformationFormat format, PdbToXmlOptions options, Int32 expectedValueSourceLine, String expectedValueSourcePath)
	at Microsoft.CodeAnalysis.Test.Utilities.PdbValidation.VerifyPdb(CompilationVerifier verifier, String qualifiedMethodName, String expectedPdb, IMethodSymbol debugEntryPoint, DebugInformationFormat format, PdbToXmlOptions options, Int32 expectedValueSourceLine, String expectedValueSourcePath)
	Emit\EditAndContinue\LocalSlotMappingTests.cs(2382,0): at Microsoft.CodeAnalysis.CSharp.EditAndContinue.UnitTests.LocalSlotMappingTests.Switch_Patterns()

33 passed, 1 failed, 0 skipped, took 7.69 seconds (xUnit.net 2.2.0 build 3444).

```

**Comment by gafter** [comment]
I've skipped this test to unblock master.

**Comment by tmat** [comment]
Fixed by #19683
